prometheus-to-sd/main.go

```diff
@@ -73,13 +73,13 @@ func main() {
 	defer glog.Flush()
 	flag.Parse()
 
-	sourceConfigs := config.SourceConfigsFromFlags(source, component, host, port, whitelisted)
-
-	gceConf, err := config.GetGceConfig(*metricsPrefix, *zoneOverride)
-	podConfig := &config.PodConfig{
+	podConfig := config.PodConfig{
 		PodId:       *podId,
 		NamespaceId: *namespaceId,
 	}
+	sourceConfigs := config.SourceConfigsFromFlags(source, component, host, port, whitelisted, podConfig)
```

Comment:
nit: podConfig should be created inside this function from flags as well.

Reply:
resolved